config/local.json

```diff
@@ -0,0 +1,18 @@
+{
+  "capabilities": {
+      "project": "Pytest Browserstack",
+      "build": "pytest-browserstack",
+      "name": "local_test",
```

Comment:
name as `BStack pytest local` (we want to keep BStack in the test name for sample repos..)

Reply:
Done https://github.com/browserstack/pytest-browserstack/pull/3/commits/f6f548fa2e0fba0f738e7e794ee9bced345e05f8